cli/edp/summarize/battery.py

```diff
@@ -0,0 +1,32 @@
+import csv
+import sys
+
+
+def summarize(filepath):
+    cycle_discharge = {}
+    with open(filepath) as fp:
+        reader = csv.reader(fp)
+
+        headers = next(reader)
+
+        cycle_index_column = headers.index('Cycle_Index')
+        discharge_capacity_column = headers.index('Discharge_Capacity')
+
+        for row in reader:
+            for i, column in enumerate(row):
```

Comment:
I think the inner loop is not needed here. However we may want to check that `cycle_index_column` and `discharge_capacity_column` don't exceed the length of the row

Reply:
Will remove